drivers/gpio/Kconfig

```
@@ -21,6 +21,12 @@ config GPIO_SHELL
 	help
 	  Enable GPIO Shell for testing.
 
+config GPIO_NAME
+	bool "Enable GPIO names"
```

As per the guidelines, please remove <word>:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe just write:
```suggestion
	bool "GPIO names"
```

Also, removing `Enable` allows the title to contain more informative text, for example by writing:
```suggestion
	bool "Display GPIO names"
```
(if I have understood the this feature correctly)